jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -120,7 +121,7 @@ pipeline {
             steps {
                 script {
                     gitCommitWhenChanges("Added New image Manifests") { msgs ->
-                        sh "scripts/process-image-manifests ${image} ${productName} ${productOperator}"
+                        sh "scripts/process-image-manifests ${image} ${productName}"
```

Comment:
This changed merged back to delorean-ews?

Reply:
Yes, we don't use it anymore but just noticed it needs removing here: https://github.com/integr8ly/integreatly-operator/blob/delorean-ews/scripts/process-image-manifests#L28 you can see that's the only entry for that env var

Reply:
Applied here: https://github.com/integr8ly/integreatly-operator/pull/468/commits/15301e2f70e038bf8c5eacbf0fe1c5d8fa132622